cfgov/v1/models/base.py

```diff
@@ -189,28 +203,8 @@ def get_breadcrumbs(self, request):
         home_page_children = request.site.root_page.get_children()
         for i, ancestor in enumerate(ancestors):
             if ancestor in home_page_children:
-                # Add top level parent page and `/process/` url segments
-                # where necessary to BAH page breadcrumbs.
-                # TODO: Remove this when BAH moves under /consumer-tools
-                # and redirects are added after 2018 homebuying campaign.
-                if ancestor.slug == 'owning-a-home':
-                    breadcrumbs = []
-                    for ancestor in ancestors[i:]:
-                        ancestor_url = ancestor.relative_url(request.site)
-                        if ancestor_url.startswith((
-                                '/owning-a-home/prepare',
-                                '/owning-a-home/explore',
-                                '/owning-a-home/compare',
-                                '/owning-a-home/close',
-                                '/owning-a-home/sources')):
-                            ancestor_url = ancestor_url.replace(
-                                'owning-a-home', 'owning-a-home/process')
-                        breadcrumbs.append({
-                            'title': ancestor.title,
-                            'href': ancestor_url,
-                        })
-                    return breadcrumbs
-                # END TODO
+                if ancestor.specific.force_breadcrumbs:
+                    return [ancestor for ancestor in ancestors[i:]]
```

Comment:
Can this be simplified to `ancestors[i:]`, and the line below to `ancestors[i + 1:]`?

Reply:
Done in 76da944!